crates/storage/src/db/lean.rs

```diff
@@ -0,0 +1,39 @@
+use std::{path::PathBuf, sync::Arc};
+
+use redb::Database;
+
+use crate::tables::lean::{
+    lean_block::LeanBlockTable, lean_state::LeanStateTable, slot_index::SlotIndexTable,
+    state_root_index::StateRootIndexTable,
+};
+
+#[derive(Clone, Debug)]
+pub struct LeanDB {
+    pub db: Arc<Database>,
+    pub data_dir: PathBuf,
```

Comment:
```suggestion
#[derive(Clone, Debug)]
pub struct LeanDB {
    pub db: Arc<Database>,
```

path isn't used in this db